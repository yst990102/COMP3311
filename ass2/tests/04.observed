5143550 Torrisi, Yen
COMP2911 17s1 Eng. Design in Computing         36 FL  fail
DB error:  unsupported operand type(s) for *: 'NoneType' and 'int'
