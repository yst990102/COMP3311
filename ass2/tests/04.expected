5143550 Torrisi, Yen
COMP2911 17s1 Eng. Design in Computing         36 FL   fail
COMP3141 17s1 Software Sys Des&Implementat'n    - AF   fail
COMP3231 17s1 Operating Systems                 - AW
COMP3151 17s2 Foundations of Concurrency       41 FL   fail
COMP4336 17s2 Mobile Data Networking           30 FL   fail
UOC = 0, WAM = 26.8
